src/Google/Ads/GoogleAds/Lib/V1/GoogleAdsClientBuilder.php

```diff
@@ -210,6 +224,19 @@ public function validate()
             throw new InvalidArgumentException('Endpoint must be a valid URL.');
         }
 
+        // For the proxy URI using filter_var is ok because the GRPC library expects the URI
+        // in a very specific format.
+        if (!empty($this->proxy) &&
+            filter_var(
+                $this->proxy,
+                FILTER_VALIDATE_URL,
+                FILTER_FLAG_SCHEME_REQUIRED | FILTER_FLAG_HOST_REQUIRED
+            ) === false) {
```

Comment:
Could you check if this is PSR2 compliant?
This line formatting smells a bit to me.

Reply:
```
./vendor/bin/phpcs src/Google/Ads/GoogleAds/Lib/V1/GoogleAdsClientBuilder.php --standard=PSR2 -np
. 1 / 1 (100%)
```